### ISSUE TITLE
Build: Support compilation of specified cc_library

### ISSUE THREAD

**Issue by ilixiaocui** [opencurve/curve#2322]
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**

The current [Makefile](https://github.com/opencurve/curve/blob/master/Makefile
) of Curve only supports the compilation of cc_binary. 

```
curve$ make
## list
Usage:
    make list stor=bs/fs
Examples:
    make list stor=bs

## build
Usage:
    make build stor=bs/fs only=TARGET dep=0/1 release=0/1 os=OS
Examples:
    make build stor=bs only=//src/chunkserver:chunkserver
    make build stor=bs only=src/* dep=0
    make build stor=fs only=test/* os=debian9
    make build stor=fs release=1
```

It is expected to support the compilation of cc_library.

**Describe the solution you'd like (描述你期望的解决方法)**

Support like: 

```
make build stor=bs only=//src/chunkserver:chunkserver_datastore
make build stor=fs only=//curvefs/src/metaserver:curvefs_metaserver
```

**Describe alternatives you've considered (描述你想到的折衷方案)**

**Additional context/screenshots (更多上下文/截图)**

**Comment by zztaki** [comment]
Hi, @ilixiaocui 
Sorry for previous communication email after I found the assignment [合并脚本 mk-tar.sh 和 mk-deb.sh](https://summer-ospp.ac.cn/org/prodetail/232990061?list=org&navpage=org) have been assigned. So I want to work on this issue and have it as my OSPP 2023 program. I will be appreciated if you could choose me to complete this assignment. 😊

Have a nice day!

**Comment by ilixiaocui** [comment]
> Hi, @ilixiaocui Sorry for previous communication email after I found the assignment [合并脚本 mk-tar.sh 和 mk-deb.sh](https://summer-ospp.ac.cn/org/prodetail/232990061?list=org&navpage=org) have been assigned. So I want to work on this issue and have it as my OSPP 2023 program. I will be appreciated if you could choose me to complete this assignment. 😊
> 
> Have a nice day!

OSPP is still in the registration stage. Multiple people can sign up for one project, and one person can also sign up for multiple projects.If you are really interested, you can try to sign up.

If you are interested in the current topic, you also need to register in the summer of open source.

**Comment by zztaki** [comment]
> > Hi, @ilixiaocui Sorry for previous communication email after I found the assignment [合并脚本 mk-tar.sh 和 mk-deb.sh](https://summer-ospp.ac.cn/org/prodetail/232990061?list=org&navpage=org) have been assigned. So I want to work on this issue and have it as my OSPP 2023 program. I will be appreciated if you could choose me to complete this assignment. 😊
> > Have a nice day!
> 
> OSPP is still in the registration stage. Multiple people can sign up for one project, and one person can also sign up for multiple projects.If you are really interested, you can try to sign up.
> 
> If you are interested in the current topic, you also need to register in the summer of open source.

Yeah, I have signed up in OSPP and choose [合并脚本 mk-tar.sh 和 mk-deb.sh](https://summer-ospp.ac.cn/org/prodetail/232990061?list=org&navpage=org) and [支持编译指定的cc_library](https://summer-ospp.ac.cn/org/prodetail/232990062?list=org&navpage=org) as alternative projects. Do I need to upload two project applications in OSPP for them and wait to be selected? But I found other issues have been assigned by others except this 😢.

**Comment by ilixiaocui** [comment]
> > > Hi, @ilixiaocui Sorry for previous communication email after I found the assignment [合并脚本 mk-tar.sh 和 mk-deb.sh](https://summer-ospp.ac.cn/org/prodetail/232990061?list=org&navpage=org) have been assigned. So I want to work on this issue and have it as my OSPP 2023 program. I will be appreciated if you could choose me to complete this assignment. 😊
> > > Have a nice day!
> > 
> > 
> > OSPP is still in the registration stage. Multiple people can sign up for one project, and one person can also sign up for multiple projects.If you are really interested, you can try to sign up.
> > If you are interested in the current topic, you also need to register in the summer of open source.
> 
> Yeah, I have signed up in OSPP and choose [合并脚本 mk-tar.sh 和 mk-deb.sh](https://summer-ospp.ac.cn/org/prodetail/232990061?list=org&navpage=org) and [支持编译指定的cc_library](https://summer-ospp.ac.cn/org/prodetail/232990062?list=org&navpage=org) as alternative projects. Do I need to upload two project applications in OSPP for them and wait to be selected? But I found other issues have been assigned to others except this 😢.

If this is the case, it is recommended to select the [cc_library that supports compilation](https://summer-ospp.ac.cn/org/prodetail/232990062?list=org&navpage=org) project and upload the corresponding design documents.

**Comment by Origin-yy** [comment]
Hello, has this task been assigned to someone else? If not, please give it to me.

**Comment by caoxianfei1** [comment]
Hi @Origin-yy , This project has been selected for OSSP https://summer-ospp.ac.cn/org/orgdetail/2994d279-0fba-47b1-9247-7b5ab604f1a8?lang=zh, but the application period has passed, if you are interested, you can continue to try it yourself.

**Comment by quas-modo** [comment]
Hello, I am the student for this issue of ospp2023, please assign this issue for me please. I will make pr in a few days.

**Comment by Wine93** [comment]
> 

Done!
